invenio_app_rdm/records_ui/views/records.py

```diff
@@ -100,6 +100,22 @@ def record_detail(pid_value, record, files, is_preview=False):
         custom_fields_ui=load_custom_fields()["ui"],
         is_preview=is_preview,
         is_draft=is_draft,
+        stats={
+            "this_version": {
+                "unique_views": "1274245",
+                "views": "14906489",
+                "unique_downloads": "72009789675",
+                "downloads": "832228",
+                "data_volume": "5000000000",
+            },
+            "all_versions": {
+                "unique_views": "1127502",
+                "views": "149081197",
+                "unique_downloads": "72009237812",
+                "downloads": "832228",
+                "data_volume": "5000000000000",
+            },
+        },
```

Comment:
Question. Why is this hardcoded?

Reply:
The backend is not ready yet. It's just a draft :)

Reply:
The actual statistics can be queried like this:
```python

def get_query(query_name):
    # TODO this could be cached in the invenio_stats extension
    from invenio_stats.proxies import current_stats

    query_config = current_stats.queries[query_name]
    query = query_config.cls(name=query_name, **query_config.params)

    return query


def get_stats(record_ui):
    # TODO this could be cached
    breakpoint()
    recid = record_ui["id"]
    parent_recid = record_ui["parent"]["id"]

    views = get_query("record-view").run(recid=recid)
    views_all = get_query("record-view-all-versions").run(parent_recid=parent_recid)
    downloads = get_query("record-download").run(recid=recid)
    downloads_all = get_query("record-download-all-versions").run(
        parent_recid=parent_recid
    )

    stats = {
        "this_version": {
            "views": views["count"],
            "unique_views": views["unique_count"],
            "downloads": downloads["count"],
            "unique_downloads": downloads["unique_count"],
            "data_volume": downloads["volume"],
        },
        "all_versions": {
            "views": views_all["count"],
            "unique_views": views_all["unique_count"],
            "downloads": downloads_all["count"],
            "unique_downloads": downloads_all["unique_count"],
            "data_volume": downloads_all["volume"],
        },
    }

    return stats
```

Then the hard-coded `stats={...}` could be replaced with `stats=get_stats(record_ui)`